distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Base_Generator.enso

```diff
@@ -104,7 +106,9 @@ lift_unary_op name function =
    - function: A function taking exactly two arguments: the generated SQL code
      for the argument of the operation, and returning the generated SQL code for
      the whole operation.
-lift_binary_op : Text -> (Builder -> Builder -> Builder) -> [Text, (Vector Builder -> Builder)]
+
+   # lift_binary_op : Text -> (Builder -> Builder -> Builder) -> [Text, (Vector Builder -> Builder)]
+lift_binary_op : Text -> (Builder -> Builder -> Builder) -> Vector
```

Comment:
Shall we do?
```suggestion
lift_binary_op : Text -> (Builder -> Builder -> Builder) -> Pair Text (Vector Builder -> Builder)
```

I know that `[x, y]` is _not_ technically a Pair, but we treat them +- interchangeably. Although that may not be great once we start checking `_ : Pair` and the value `[a, b]` will fail that check.